src/blueprints/tv2/tv2-on-timeline-generate-service.ts

```diff
@@ -281,6 +295,51 @@ export class Tv2OnTimelineGenerateService implements BlueprintOnTimelineGenerate
     }
   }
 
+  private updateTriCasterProgramWithMediaPlayer(timelineObject: Tv2BlueprintTimelineObject, mediaPlayer: Tv2MediaPlayer): void {
+    if (timelineObject.content.deviceType !== DeviceType.TRICASTER || timelineObject.content.type !== TriCasterType.ME) {
+      return
+    }
+    const triCasterMeTimelineObject: TriCasterMixEffectTimelineObject = timelineObject as TriCasterMixEffectTimelineObject
+    if (triCasterMeTimelineObject.content.me.type !== TriCasterMixEffectContentType.PROGRAM) {
```

Comment:
Can't `isTriCasterTimelineObject` be used for both conditions here?

Reply:
Updated